content/v2.0/query-data/influxql.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Query data with InfluxQL
+description: >
+  Use the [InfluxDB 1.x `/query` compatibility endpoint](/v2.0/reference/api/influxdb-1x/query)
+  to query data in InfluxDB 2.0 with **InfluxQL**.
+weight: 102
+v2.0/tags: [influxql, query]
+menu:
+  v2_0:
+    name: Query with InfluxQL
+    parent: Query data
+products: [cloud]
+related:
+  - /v2.0/reference/api/influxdb-1x/
+  - /v2.0/reference/api/influxdb-1x/query
+  - /v2.0/reference/api/influxdb-1x/dbrp
+---
+
+Use the [InfluxDB 1.x `/query` compatibility endpoint](/v2.0/reference/api/influxdb-1x/query)
+to query data in InfluxDB 2.0 with **InfluxQL**.
+The [InfluxDB 1.x compatibility API](/v2.0/reference/api/influxdb-1x/) supports
+all InfluxDB 1.x client libraries and integrations in InfluxDB 2.0.
+
+Provide the following:
+
+- InfluxDB [authentication token](/v2.0/security/tokens/)
+  _(See [compatibility API authentication](/v2.0/reference/api/influxdb-1x/#authentication))_
+- **db query parameter**: 1.x database to query
+- **rp query parameter**: 1.x retention policy to query
+- **q query parameter**: InfluxQL query
+
+{{% note %}}
+**URL-encode** the InfluxQL query to ensure it's formatted correctly when submitted to InfluxDB.
+{{% /note %}}
+
+```sh
+curl -G http://localhost:9999/query?db=db&rp=rp \
```

Comment:
since this is cloud only (for the moment), using localhost:9999 seems weird

Reply:
I updated this so now we can have code blocks for cloud-only examples. They use a placeholder URL, `https://cloud2.influxdata.com` that gets updated like the other code blocks. Right now, it defaults to AWS US West if another cloud region isn't selected. If another cloud region is selected, it uses that URL.